sunpy/spectra/spectrum.py

```diff
@@ -81,6 +84,9 @@ def plot(self, axes=None, **matplot_args):
 
         return lines
 
+            
+
+
```

Comment:
They came because in the earlier commits I was making the check in a different function. I'll remove them in the final commit, along with any other changes that might be needed.